exercises/concept-wip/wellingtons-weather-station/.meta/src/exemplar.cr

```diff
@@ -0,0 +1,17 @@
+class Temperature
```

Comment:
```suggestion
module Temperature
  extend self
  
```

Reply:
You are absolutely right that modules are more correct.

But my philosophy is to not introduce syntax without explaining it.
I had actually structured the first exercises by using: `self.<method_name>` but Erik recommended doing it like it is now. Which the Ruby track also does. The difference is that the ruby track moves very quickly into oop. I think 3 of their concept exercises are non oop. I think around 1/6 of crystals practice exercises are oop which I think is drastically lower than the ruby track. This syllabus moves slower into oop and lets that appear later. After number types in fact. My goal is to introduce using self.<method_name> with classes in the class concept.

Reply:
That's cool.

I had a quick browse through Ruby's syllabus. They talk about classes in the Basics concept, and only the Modules concept exercise uses a module.

I think I was getting a bit confused with my own Ruby solutions where I use modules pervasively.